go.mod

```diff
@@ -14,17 +14,17 @@ require (
 	github.com/onsi/gomega v1.30.0
```

Comment:
We prefer to commit `go mod vendor` in the final commit in the PR, because if you `go mod vendor` and add it in 1 commit, it makes it hard to review (getting rid of `go mod vendor` won't happen any time soon)